src/dev/build/tasks/create_cdn_assets_task.ts

```diff
@@ -23,10 +23,11 @@ export const CreateCdnAssets: Task = {
 
   async run(config, log, build) {
     const buildSource = build.resolvePath();
-    const buildNum = config.getBuildNumber();
+    // TODO: update to buildShaShort after https://github.com/elastic/kibana/pull/175898
+    const buildSha = config.getBuildSha().substring(0, 12);
```

Comment:
Can we update this since the PR is merged?

Reply:
Thanks, I pushed https://github.com/elastic/kibana/pull/176186/commits/eb86b182ff33098421b34515bfba68e420fb1241